test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -667,7 +667,7 @@ ZoneId=$FileType
                 $testData += @(
                     @{
                         module = $PSHomeUntrustedModule
-                        error = $null
+                        error = $error
```

Comment:
not sure about this - is this just whatever is there?

Reply:
We should have been expecting an error defined at line # 657. I will change the variable name to make it clearer.